### DISCUSSION TITLE
Request to Conduct User Research for Master Thesis

### DISCUSSION THREAD

**Discussion by niklasmamuzic** [jamulussoftware/jamulus#3357] [General]
Hi Jamulus Team,

My name is Niklas, and I’m currently pursuing my Master's degree in Management at the Technical University of Munich. I’m reaching out because I’m interested in conducting user research on Jamulus as part of my Master thesis, which focuses on evaluating the commercial viability of platforms for real-time online music collaboration.

Volker Fischer, who kindly guided me to this forum, mentioned that this would be the right place to discuss this idea.

For my thesis, I aim to understand the needs, preferences, and experiences of Jamulus users. The insights gained from this research could help identify potential improvements and opportunities for the platform's growth. Specifically, I plan to design a survey among Jamulus users as well as some expert interviews with you.

Would it be possible to get your approval and support for this research? I believe the results could be valuable for both my academic work and the ongoing development of Jamulus.

Thank you for considering my request. I’m happy to provide more details or answer any questions you might have.

Best regards,
Niklas

**Comment by chrisrimple** [comment]
Given that Jamulus is maintained by a group of volunteers and has no "owner", I don't know that "approval" for your survey is needed.  After all, you're just collecting data from Jamulus users, which anyone could do without even mentioning their actions in this forum.  But thank you for making everyone aware.  Others have queried Jamulus users for similar surveys in the past.

Apart from joining public servers when they are populated with musicians and asking for those musicians to participate, you can also find many Jamulus users via these Facebook Groups...

https://www.facebook.com/groups/jamulus/
https://www.facebook.com/groups/1252784948406794/
https://www.facebook.com/groups/JamulusWorldJam/
https://www.facebook.com/groups/607066493580615/
https://www.facebook.com/groups/752259165602067/
https://www.facebook.com/groups/wjmidwest/
https://www.facebook.com/groups/jamulusevents/
https://www.facebook.com/groups/jamuluschoralcommunity/
https://www.facebook.com/groups/jam.sur.internet/
https://www.facebook.com/groups/330658772319094/
https://www.facebook.com/groups/jamulus.th/

Good luck!

**Comment by ann0see** [comment]
That's certainly interesting. I think it makes sense to learn about the nature of free and open source projects too to understand Jamulus (take a little time, not too much, since you can get deep into the rabbit hole). (You can check out the Free software foundation's website on fsf.org and of course https://de.m.wikipedia.org/wiki/Free_Software_Foundation and https://opensource.org/osd they aren't the same philosophy and we probably don't follow those rigorously but it's good knowing it)

For some feature requests: You can dig around this forum too. 

Also, there are some interesting publications concerning Jamulus floating around in the internet. 

> Open Educational Resources und Open Source–Beiträge zu einer nachhaltigen Infrastruktur für digitale Lernumgebungen

> Svenja Müller, Peramachanahalli S Ramkumar, Jörg Rapp, Engelbert Niehaus


Check google scholar for Jamulus.

**Comment by ann0see** [comment]
> I don't know that "approval" for your survey is needed

Certainly not. Everything is freely available anyway.

**Comment by alexbuckland** [comment]
Hi, I’m the co-founder of melomax.live. We run a popular jamulus hosting service. Happy to help you out if we can 
- Alex
